src/pool-bin/interfaces/IBinPositionManager.sol

```diff
@@ -9,7 +9,7 @@ import {IPositionManager} from "../../interfaces/IPositionManager.sol";
 
 interface IBinPositionManager is IPositionManager {
     error IdOverflows(int256);
-    error IdDesiredOverflows(uint24);
+    error IdSlippageCaught(uint24);
```

Comment:
Should we include the params.idSlippage and activeId as well per the report ? 

```

if (params.activeIdDesired + params.idSlippage < activeId) revert IdSlippageCaught(params.activeIdDesired, params.idSlippage, activeId);

if (params.activeIdDesired - params.idSlippage > activeId) revert IdSlippageCaught(params.activeIdDesired, params.idSlippage, activeId);

```

Reply:
agree. please ensure these 3 param are in the event: `(params.activeIdDesired, params.idSlippage, activeId)`